docs/development-environment.md

```diff
@@ -194,7 +200,7 @@ We recommend usage of [nvm](https://github.com/nvm-sh/nvm/) for managing differe
 run this command before building again. Otherwise you may experience errors on the command line while trying to build.
 
 ```sh
-pnpm distclean
+cd projects/plugins/jetpack && pnpm run clean
```

Comment:
`jetpack clean --dist` also removes all node_modules and vendor folders for all projects in the monorepo. Not sure if that fills the same need as this command, but if it does, we could suggest that instead.

Reply:
This is perfect, that's just what we'd need here.
42c67f7e1edc789076d5c078ed9e3c50b70adebf